src/github.py

```diff
@@ -75,8 +75,9 @@ def __init__(self, key: str = None, repo: str = None, org: str = None, content:
         self.github_key = content['number']
         self.jira_key = self.get_jira_equivalent()
         self.summary = content['title']
-        self.created = datetime.datetime.strptime(content['created_at'].split('Z')[0], '%Y-%m-%dT%H:%M:%S')
-        self.updated = datetime.datetime.strptime(content['updated_at'].split('Z')[0], '%Y-%m-%dT%H:%M:%S')
+        # Get datetime objects from timestamp strings and adjust for time zone
+        self.created = datetime.datetime.strptime(content['created_at'].split('Z')[0], '%Y-%m-%dT%H:%M:%S') - datetime.timedelta(hours=7)
```

Comment:
Could you please elaborate on the purpose of adding delta time (i.e., did you find a use case where the previous code ran into problems)? Do we think that someone not in PST might post? In that case it could be any deviation from PST, depending on the user's timezone. 
Besides, if you use GitHub what timezone will your timestamp be in? If two people were to simultaneously  commit in two different timezones, are the timestamps different?

Reply:
Yes, Jira returns timestamps in local time, and ZenHub/GitHub return timestamps in UTC. Unfortunately Jira does not say what time zone it is in, so I am assuming all are in PST. You're right there are problems here. Updated to use pytz to accurately compare time zones and account for daylight savings.